Firestore/core/src/remote/serializer.h

```diff
@@ -189,7 +189,8 @@ class Serializer {
   EncodeListenRequestLabels(const local::TargetData& target_data) const;
 
   static pb_bytes_array_t* EncodeFieldPath(const model::FieldPath& field_path);
-  static model::FieldPath DecodeFieldPath(const pb_bytes_array_t* field_path);
+  static util::StatusOr<model::FieldPath> DecodeFieldPath(
```

Comment:
Similarly, this should take a `Reader` and avoid returning `StatusOr`.

Reply:
Thanks